### ISSUE TITLE
ignore unknown sections in select.def

### ISSUE THREAD

**Issue by lokpro** [ikemen-engine/Ikemen-GO#356]
Hi,
ref to the example, myChar2~5 are ignored in Mugen. Ikemen-GO reads all lines from myChar0 to 5 .
I have added several sections to quickly switch between my groups of characters.
```
; example select.def
[Characters]
myChar0
myChar1
[Characters_A]
myChar2
myChar3
[Characters_B]
myChar4
myChar5
[ExtraStages]
```

---

I have added these two lines in _main.lua_, and it does what I want.
```
elseif lineCase:match('^%s*%[%s*storymode%s*%]') then
  row = 0
  section = 4 // ln: 1872

// two lines added
elseif lineCase:match('^%s*%[%w+%]$') then 
  section = -1

elseif section == 1 then --[Characters] // ln: 1873
```

https://github.com/Windblade-GR01/Ikemen-GO/blob/7b57d90abc4b8468224caea418d3d973045d17ff/external/script/main.lua#L1849-L1873

**Comment by K4thos** [comment]
will be fixed in 0.98 release, thanks

**Comment by lokpro** [comment]
I  don't know lua well. I do Javascript. Be sure to check my code :)
